conn_test.go

```diff
@@ -1527,9 +1529,9 @@ func TestExecWithQueryExecTimeoutSet(t *testing.T) {
 
 	_, err := conn1.Exec("SELECT pg_sleep(2)")
 	if err == nil {
-		t.Fatal("Expected Exec to fail with 'conn is dead', instead it did not")
+		t.Fatal("Expected Exec to fail with 'use of closed network connection', instead it did not")
 	}
-	if !strings.Contains(err.Error(), "conn is dead") {
+	if !strings.Contains(err.Error(), "use of closed network connection") {
```

Comment:
We do log timeout occurrence.
But take a look at the last commit plz - it has some changes that allows us to see the timeout in the returned error.